packages/next/src/bin/next.ts

```diff
@@ -198,6 +199,34 @@ program
   })
   .usage('[directory] [options]')
 
+program
```

Comment:
Does it make sense to have a `--no-build` option in case you already built with all the right flags to emit the analyzer output?